bundles/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/SoulissBindingConstants.java

```diff
@@ -0,0 +1,218 @@
+    public static final ThingTypeUID T51_THING_TYPE = new ThingTypeUID(BINDING_ID, T51);
+    public static final ThingTypeUID T52_THING_TYPE = new ThingTypeUID(BINDING_ID, T52);
+    public static final ThingTypeUID T53_THING_TYPE = new ThingTypeUID(BINDING_ID, T53);
+    public static final ThingTypeUID T54_THING_TYPE = new ThingTypeUID(BINDING_ID, T54);
+    public static final ThingTypeUID T55_THING_TYPE = new ThingTypeUID(BINDING_ID, T55);
+    public static final ThingTypeUID T56_THING_TYPE = new ThingTypeUID(BINDING_ID, T56);
+    public static final ThingTypeUID T57_THING_TYPE = new ThingTypeUID(BINDING_ID, T57);
+    public static final ThingTypeUID T58_THING_TYPE = new ThingTypeUID(BINDING_ID, T58);
+    public static final ThingTypeUID T61_THING_TYPE = new ThingTypeUID(BINDING_ID, T61);
+    public static final ThingTypeUID T62_THING_TYPE = new ThingTypeUID(BINDING_ID, T62);
+    public static final ThingTypeUID T63_THING_TYPE = new ThingTypeUID(BINDING_ID, T63);
+    public static final ThingTypeUID T64_THING_TYPE = new ThingTypeUID(BINDING_ID, T64);
+    public static final ThingTypeUID T65_THING_TYPE = new ThingTypeUID(BINDING_ID, T65);
+    public static final ThingTypeUID T66_THING_TYPE = new ThingTypeUID(BINDING_ID, T66);
+    public static final ThingTypeUID T67_THING_TYPE = new ThingTypeUID(BINDING_ID, T67);
+    public static final ThingTypeUID T68_THING_TYPE = new ThingTypeUID(BINDING_ID, T68);
+    public static final ThingTypeUID TOPICS_THING_TYPE = new ThingTypeUID(BINDING_ID, TOPICS);
+
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections
+            .unmodifiableSet(Stream.of(GATEWAY_THING_TYPE, T11_THING_TYPE, T12_THING_TYPE, T13_THING_TYPE,
+                    T14_THING_TYPE, T16_THING_TYPE, T18_THING_TYPE, T19_THING_TYPE, T1A_THING_TYPE, T21_THING_TYPE,
+                    T22_THING_TYPE, T31_THING_TYPE, T41_THING_TYPE, T42_THING_TYPE, T51_THING_TYPE, T52_THING_TYPE,
+                    T53_THING_TYPE, T54_THING_TYPE, T55_THING_TYPE, T56_THING_TYPE, T57_THING_TYPE, T58_THING_TYPE,
+                    T61_THING_TYPE, T62_THING_TYPE, T63_THING_TYPE, T64_THING_TYPE, T65_THING_TYPE, T66_THING_TYPE,
+                    T67_THING_TYPE, T68_THING_TYPE, TOPICS_THING_TYPE).collect(Collectors.toSet()));
+
+    // List of all Channel ids
+    public static final String ONOFF_CHANNEL = "onoff";
+
+    public static final String PULSE_CHANNEL = "pulse";
+    public static final String SLEEP_CHANNEL = "sleep";
+    public static final String AUTOMODE_CHANNEL = "automode";
+    public static final String STATEONOFF_CHANNEL = "stateOnOff";
+    public static final String STATEOPENCLOSE_CHANNEL = "stateOpenClose";
+    public static final String ROLLERSHUTTER_CHANNEL = "rollershutter";
+    public static final String ROLLERSHUTTER_STATE_CHANNEL_CHANNEL = "rollershutterstate";
+    public static final String ROLLERSHUTTER_STATE_NUMBER_CHANNEL = "rollershutter_state_number";
```

Comment:
This does not appear to show up in the thing-types.xml file?

Reply:
commented last hilighted line future use.